Suggested edits on Vault use cases (#16502)
hashicorp/vault: website/content/docs/use-cases.mdx

## Secrets Management

> The adoption of Cloud means organizations shift away from static infrastructure to now provisioning and managing dynamic infrastructure. This implies distribution of services, embracing ephemerality and immutability, and the ability to deploy onto multiple target environments. 

Reviewer: ```suggestion
The adoption of the cloud means organizations shift away from static infrastructure to now provisioning and managing dynamic infrastructure. This implies distribution of services, embracing ephemerality and immutability, and the ability to deploy onto multiple target environments. 
```

Managing secrets in the cloud can add complexity as organizations look to adopt more modern
security practices and posture. When choosing the right secrets management solution there are several options available depending on security and deployment requirements for workflows, teams, systems and infrastructure.

Vault's [secret management](/docs/secrets) solution centrally manages and enforces access to secrets and systems based on trusted sources of application and user identity. 

### Static Secrets

> Vault safely store credentials, keys, and other types of secrets without deplying and managing a complex key management infrastructure.

Reviewer: ```suggestion
Vault safely store credentials, keys, and other types of secrets without deploying and managing a complex key management infrastructure.
```

> - Read the [Versioned Key/Vault Secrets Engine](https://learn.hashicorp.com/tutorials/vault/versioned-kv?in=vault/secrets-management) tutorial and learn how a versioned key-value secrets engine protects your static secrets.

Reviewer: ```suggestion
- Refer to the [Versioned Key/Vault Secrets Engine](https://learn.hashicorp.com/tutorials/vault/versioned-kv?in=vault/secrets-management) tutorial and learn how a versioned key-value secrets engine protects your static secrets.
```

### Dynamic Secrets

Secrets in plain text may accidentally be left in an application or with an individual long after the intended period of access. Vault provides a unique ability to create, rotate, and revoke secrets dynamically.

> - Read the [Dynamic Secrets: Database Secrets Engine](https://learn.hashicorp.com/tutorials/vault/database-secrets) tutorial and learn how Vault can dynamically manage your database credentials.

Reviewer: ```suggestion
- Refer to the [Dynamic Secrets: Database Secrets Engine](https://learn.hashicorp.com/tutorials/vault/database-secrets) tutorial and learn how Vault can dynamically manage your database credentials.
```

In addition to database credential management, Vault can manage your Active Directory accounts, SSH keys, PKI certificates and more. Visit the [Secrets Management](https://learn.hashicorp.com/collections/vault/secrets-management) tutorial series to learn more about secrets management using Vault. 

## Data Encryption

Many organizations seek solutions to encrypt/decrypt application data within a cloud or multi-datacenter environment; deploying cryptography and maintaining a complex key management infrastructure can be expensive and challenging to develop. Vault provides [encryption as a service](/docs/secrets/transit) with centralized key management to simplify encrypting data in transit and stored across clouds and datacenters. Vault can encrypt/decrypt data stored elsewhere, essentially allowing applications to encrypt their data while storing it in the primary data store. Vault's security team manages and maintains the responsibility of the data encryption within the Vault environment, allowing developers to focus solely on encrypting/decrypting data as needed.

### Resources

> Try our [Encryption as a Service: Transit Secrets Engine](https://learn.hashicorp.com/collections/vault/encryption-as-a-service) to learn the essential workings of the Transit secrets engine handles cryptographic functions on data in-transit.

Reviewer: ```suggestion
Try our [Encryption as a Service: Transit Secrets Engine](https://learn.hashicorp.com/collections/vault/encryption-as-a-service) tutorial to learn the essential workings of the Transit secrets engine handles cryptographic functions on data in-transit.
```
